client/python/gradio_client/serializing.py

```diff
@@ -164,7 +167,10 @@ def serialize(
         """
         if x is None or x == "":
             return None
-        filename = str(Path(load_dir) / x)
+        if utils.is_valid_url(x):
```

Comment:
There was a bug where urls were not being downloaded during pre-processing

Reply:
Nice catch